IFC4x3/Properties/i/IsElectricalyInsulated_1HhghvvaLAvfFIOypPw7B9/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="IsElectricalyInsulated_1HhghvvaLAvfFIOypPw7B9" Name="IsElectricalyInsulated" UniqueId="51aeaaf9-e645-4ae6-93d2-63ccd9e872c9" PrimaryDataType="IfcBoolean" />
```

Comment:
Seems both possible by its design or by surrounding elements, so I still put it in Pset_TrackElementTypeSleeper as it could be override in occurrence level.